eos-add-flatpak-gnome-repos

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+# Copyright (C) 2016 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+# Add the stable GNOME flatpak repositories after an OS upgrade
+
+TMPDIR=`mktemp -d`
```

Comment:
You can also add your paranoia here: `trap '[ -n "$TMPDIR" ] && rm -rf "$TMPDIR"' EXIT`. I forgot about quoting this correctly, too.